centreon/www/class/centreon-clapi/centreonDependency.class.php

```
@@ -291,15 +291,17 @@ public function addchild($parameters): void
      *
      * @param $filterName
      *
-     * @return void
+     * @return bool
      */
-    public function export($filterName = null): void
+    public function export($filterName = null): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$filterName'. (undefined)

(UnusedFormalParameter)